src/app/static/src/app/main.ts

```diff
@@ -6,9 +6,11 @@ import {
     surveyAndProgram,
     SurveyAndProgramDataState
 } from "./store/surveyAndProgram/surveyAndProgram";
+import {password, initialPasswordState, PasswordState} from "./store/password/password";
 
 export interface RootState {
     version: string;
+    password: PasswordState,
```

Comment:
Given the proposed separateness of the main app from the pw logic this wouldn't need to be here. Indeed the set password app would be much simpler - no modules, the root state for that would just be the password state.